src/test/java/Homework17.java

```diff
@@ -0,0 +1,67 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+
+
+@Test
+public void addSongToPlaylist(){
+openLoginUrl("https://qa.koel.app/");
+provideEmail("momo.carlos5@gmail.com");
+providePassword("te$t$tudent");
+cliquesubmit();
+SearchSongs("BossStatus");
+clickViewAll();
+addSongToMyPlaylist();
+
+
+    
+        }
+
+    private void addSongToMyPlaylist() {
+        driver.findElement(By.xpath("//*[@id=\"songResultsWrapper\"]/div/div/div[1]/table/tr")).click();
+        driver.findElement(By.xpath("//*[@id=\"songResultsWrapper\"]/header/div[3]/span/button[2]")).click();
+        driver.findElement(By.xpath("//*[@id=\"songResultsWrapper\"]/header/div[3]/div/section[1]/ul/li[5]")).click();
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

_For example, here is the locator (Copy > Copy Xpath):_  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**
For example, here is the correct locator: ` //a[@class='songs']`

It is important to mention that the following principle must be held in deciding locators: the locator must be

- unique
- readable 
- maintainable